package.json

```diff
@@ -25,7 +25,7 @@
     "build:web-modules": "node ./bin/install-web-modules.js ./dist/js/web_modules preact preact-router",
     "build:static": "node ./bin/copy.js ./sources/static ./dist/",
     "build:ts": "npx ttsc --build tsconfig.json",
-    "build:scss": "npx node-sass ./sources/scss/manifest.scss dist/css/main.css --output-style compressed",
+    "build:scss": "npx sass ./sources/scss/manifest.scss dist/css/main.css",
```

Comment:
Is sass compressing the files by default?

Reply:
Good catch - fixed. The main reason why sass is just a little faster and works on windows os also